CHANGELOG

```diff
@@ -1,6 +1,10 @@
 django-plans changelog
 ======================
 
+1.1.0 (unreleased)
+------------------
+* Add `AbstractOrder.return_order`
```

Comment:
`AbstractOrder.return_order()` to be clear that it is a function.

Reply:
Changed